src/templates/MeetingView/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+
+import GearImage from '../../../images/gear.png';
+import { DropdownButton, DropdownContainer, DropdownContentWrapper, ImageWrapper } from './styled';
+
+interface Props {
+  onClickConfirmButton: () => void;
+  onClickEditButton: () => void;
+}
+
+export function Dropdown(props: Props) {
+  const { onClickConfirmButton, onClickEditButton } = props;
+
+  const [isShow, setShow] = useState(false);
+
+  useEffect(() => {
+    if (isShow) {
```

Comment:
```suggestion
    if (!isShow) {
```
혹시 조건문이 반대로 걸린게 아닌가요?
열려 있을때 return이 되면 안될것 같아요!